src/app/pages/rpc-send-transfer/use-rpc-send-transfer-actions.tsx

```diff
@@ -58,6 +59,10 @@ export function useRpcSendTransferActions() {
 
         await broadcastTx({
           tx: tx.hex,
+          // TODO: Add map for inscriptions after query hooks are relocated. This conditional
```

Comment:
Do we have a task for this we can link?

Reply:
Made an issue here: https://linear.app/leather-io/issue/LEA-2398/refactor-utxo-inscription-check-and-filtering